src/app/api/snippets/[language]/[category]/[snippetTitle]/route.ts

```
@@ -0,0 +1,41 @@
+
+  const data = JSON.parse(fs.readFileSync(filePath, "utf8"));
+  const categoryData = data.categories.find(
+    (cat: any) => slugify(cat.categoryName) === slugify(category)
+  );
+
+  if (!categoryData) {
+    return NextResponse.json({ error: "Category not found" }, { status: 404 });
+  }
+
+  const snippet = categoryData.snippets.find(
+    (snip: any) => slugify(snip.title) === slugify(snippetTitle)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)